.circleci/config.yml

```diff
@@ -264,12 +264,53 @@ jobs:
     - run: |
         sudo $GOPATH/bin/registry serve integration/docker-registry.yaml > /dev/null 2>&1 &
         PORT=4569 http-echo-server > /dev/null 2>&1 &
-        $GOPATH/bin/ginkgo -p -stream integration/init
+        $GOPATH/bin/ginkgo -p -stream --focus='shipinit' integration/init
     - save_cache:
         paths:
         - /tmp/go/cache
         key: ship-integration-test-build-cache-init-{{ epoch }}
 
+
+  integration_init_edit:
+    machine: true
+    working_directory: /home/circleci/go/src/github.com/replicatedhq/ship
+    environment:
+      GOPATH: /home/circleci/go
+      GO_SHA256SUM: 2a3fdabf665496a0db5f41ec6af7a9b15a49fbe71a85a50ca38b1f13a103aeec
+      GO: /usr/local/go/bin/go
+      GOCACHE: "/tmp/go/cache"
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - ship-integration-test-build-cache-init
+      - run: |
```

Comment:
I see what you mean about refactoring these. Not a blocker for this PR, but maybe as a stopgap we could start with a YAML anchor for this run step?

Reply:
Yeah, that feels like it might be a good idea
But I think moving away from 'machine' to one of our builder images or just 'go' would be the better long-term choice